app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::ItemsController < ApiController
+  before_action :authenticate_user
+
+  def create
+    list = List.find(params[:list_id])
+    item = list.items.build(item_params)
+
+    if item.save
+      render json: item, status: :success
+    else
+      render json: { errors: item.errors.full_messages }, status: :unprocessabel_entity
```

Comment:
There's a typo at the end of this line, it should be `status: :unprocessable_entity`.